static/elements/chromedash-feature-table.js

```diff
@@ -234,29 +264,87 @@ class ChromedashFeatureTable extends LitElement {
     return nothing;
   }
 
+  getEarliestReviewDate(feature) {
+    const configs = this.configs[feature.id];
+    const allDates = configs.map(c => c.next_action).filter(d => d);
+    if (allDates.length > 0) {
+      allDates.sort();
```

Comment:
Is this a native `sort()` function? I couldn't find the API for it

Reply:
Yeah, it is just the native sort() function:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

One thing to keep in mind about JS array sort() is that it sorts everything as strings, unless a conversion function is supplied.  That can give unexpected results when trying to sort numbers.  However, in this case, the data are dates in format YYYY-MM-DD, so sorting them as strings seems fine.